ncard_app/static/table.css

```diff
@@ -0,0 +1,24 @@
+#ncard_table {
+    overflow-x: scroll;
+}
+#ncard_table table {
+    table-layout: fixed;
+}
+#ncard_table th {
+    width: 200px;
+    
+}
```

Comment:
This spacing is inconsistent. There should be no line break inside the selector, and one line break between selectors.